lib/src/facts/linux/virtualization_resolver.cc

```diff
@@ -117,6 +117,11 @@ namespace facter { namespace facts { namespace linux {
                 return true;
             }
             // Take the first line that isn't an error/warning
+            // unless it's "xen", in which case we expect a second
+            // line with more useful information
```

Comment:
After re-reading the ticket and the virt-what man page, I agree that we're safe here, so let's not add YAGNI complexity to the code.